mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
Yes, if we don't use csc.exe from this Roslyn folder in msbuild then we shouldn't need all the Microsoft.CodeAnalysis.*.dll files etc.

I thought VBCSCompiler.exe is the compiler server which we don't support yet? I guess we could not copy it as well.